src/envo/envo-edit.owl

```diff
@@ -28119,6 +28142,155 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000032> <http://purl.obolibrar
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000038> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> <http://purl.obolibrary.org/obo/ENVO_00001995>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000038> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> <http://purl.obolibrary.org/obo/ENVO_00002008>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000038> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> <http://purl.obolibrary.org/obo/ENVO_03000021>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000038> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002353> <http://purl.obolibrary.org/obo/ENVO_03000020>))
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000039> (cryoconite hole)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "http://glaciers.pdx.edu/Projects/Antarctica/CryoconiteHoles/Cryo_main.html") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000039> "A vertical hole within an ice mass, formed by ice melt associated with local temperature increases, due to increased absorption of solar radiation by cryoconite deposits.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000116> <http://purl.obolibrary.org/obo/ENVO_03000039> "The general semantics of depressions are still to be worked out see  https://github.com/EnvironmentOntology/envo/issues/486")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000117> <http://purl.obolibrary.org/obo/ENVO_03000039> "http://orcid.org/0000-0002-3410-4655")
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_03000039> "This class doesn't refer to the site but instead the depression or hole and therefore the adjacency to the cryoconite itself.")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03000039> "cryoconite hole"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000039> <http://purl.obolibrary.org/obo/ENVO_00000309>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000039> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> <http://purl.obolibrary.org/obo/ENVO_03000038>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000039> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002353> <http://purl.obolibrary.org/obo/ENVO_01000721>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000039> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002353> <http://purl.obolibrary.org/obo/ENVO_03000020>))
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000040> (melt pond)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Melt_pond") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000040> "A small water body, which forms via a melting processes occuring within a continuous mass of ice such a glacier or on sea ice.")
```

Comment:
@jzrapp I was referring to glaciers or sea ice masses when I found these definitions. I will, however, take a look at the paper you suggested and see if I can refine the definitions. It would be most appreciate if you could comment again I make some changes, or if you have time, on any of my other terms.